ocs_samples/basic_samples/OmfIngress/DotNet/readme.md

```diff
@@ -0,0 +1,173 @@
+.NET Samples 
+============
+
+Building a Client with the Ingress Client Libraries
+---------------------------------------------------
+
+The sample described in this section makes use of the OSIsoft Ingress Client Libraries. When working in .NET, 
+it is recommended that you use these libraries. The libraries are available as NuGet packages. The packages used are:
+
+* OSIsoft.Omf
+* OSIsoft.OmfIngress
+* OSIsoft.Identity.AuthenticationHandler
+
+The libraries offer a framework of classes that make client development easier.
+
```

Comment:
Can you include steps to run this sample?  You can see the other dotnet samples for inspiration, but we should include command line at the least and if you want to, you can include your favorite IDE.